modules/sw360/sw360-workflow/src/test/java/org/eclipse/sw360/antenna/sw360/SW360MetaDataUpdaterTest.java

```diff
@@ -275,7 +275,60 @@ public void testGetOrCreateReleaseFoundByExternalIDs() {
+    public void testGetOrCreateReleaseFoundByExternalIDsWithArtifactPrecedence() {
+        SW360SparseRelease sparseRelease = new SW360SparseRelease();
+        SW360Release foundRelease = new SW360Release();
+        SW360Release queryRelease = new SW360Release();
+        SW360Release patchedRelease = new SW360Release();
+        Map<String, String> extIDs = Collections.singletonMap("foo", "bar");
+        final String queryCopyright = "(C) Test copyright query";
+        final String foundCopyright = "(C) Test copyright found";
+        queryRelease.setExternalIds(extIDs);
+        queryRelease.setCopyrights(queryCopyright);
+        foundRelease.setCopyrights(foundCopyright);
+        when(releaseClientAdapter.getSparseReleaseByExternalIds(extIDs)).thenReturn(Optional.of(sparseRelease));
+        when(releaseClientAdapter.enrichSparseRelease(sparseRelease)).thenReturn(Optional.of(foundRelease));
+        when(releaseClientAdapter.updateRelease(any()))
+                .thenAnswer((Answer<SW360Release>) invocationOnMock -> {
+                    SW360Release rel = invocationOnMock.getArgument(0);
+                    assertThat(rel.getExternalIds()).isEqualTo(extIDs);
+                    assertThat(rel.getCopyrights()).isEqualTo(queryCopyright);
+                    return patchedRelease;
+                });
+
+        assertThat(metaDataUpdater.getOrCreateRelease(queryRelease, true, true)).isEqualTo(patchedRelease);
+    }
+
+    @Test
+    public void testGetOrCreateReleaseFoundByExternalIDsWithArtifactPrecedenceAndLowerClearingState() {
+        SW360SparseRelease sparseRelease = new SW360SparseRelease();
+        SW360Release foundRelease = new SW360Release();
+        SW360Release queryRelease = new SW360Release();
+        SW360Release patchedRelease = new SW360Release();
+        Map<String, String> extIDs = Collections.singletonMap("foo", "bar");
+        final String queryCopyright = "(C) Test copyright query";
+        final String foundCopyright = "(C) Test copyright found";
+        queryRelease.setExternalIds(extIDs);
+        queryRelease.setCopyrights(queryCopyright);
+        foundRelease.setCopyrights(foundCopyright);
+        foundRelease.setClearingState("OSM_APPROVED");
+        when(releaseClientAdapter.getSparseReleaseByExternalIds(extIDs)).thenReturn(Optional.of(sparseRelease));
+        when(releaseClientAdapter.enrichSparseRelease(sparseRelease)).thenReturn(Optional.of(foundRelease));
+        when(releaseClientAdapter.updateRelease(any()))
+                .thenAnswer((Answer<SW360Release>) invocationOnMock -> {
+                    SW360Release rel = invocationOnMock.getArgument(0);
+                    assertThat(rel.getExternalIds()).isEqualTo(extIDs);
+                    assertThat(rel.getCopyrights()).isEqualTo(foundCopyright);
+                    return patchedRelease;
+                });
+
+        assertThat(metaDataUpdater.getOrCreateRelease(queryRelease, true, true)).isEqualTo(patchedRelease);
```

Comment:
A lot of code in those tests is duplicated. Maybe you can refactor the duplicates into methods.